docs/plans/2026-04-23-fix-loop/capability-gaps.md

```
@@ -13,15 +13,15 @@
 | Bucket | Count | Principles |
 |--------|-------|-----------|
 | Migrated (match-only, no guard suppression) | 8 | addition-overflow, subtraction-underflow, multiplication-overflow, find-undefined-result, match-null-result, split-empty-string, falsy-default, empty-collection-loop |
-| Migrated (guard-aware / structurally complete) | 3 | reduce-no-initial, throw-uncaught, unguarded-await (partial — over-matches) |
+| Migrated (guard-aware / structurally complete) | 3 | reduce-no-initial, throw-uncaught, unguarded-await (partial (over-matches) |)
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)